shell/Dockerfile

```diff
@@ -16,8 +16,7 @@
 FROM hyperledger/sawtooth-shell:nightly
 
 # Install Python, Node.js, and Ubuntu dependencies
-RUN echo "deb http://repo.sawtooth.me/ubuntu/1.0/stable bionic universe" >> /etc/apt/sources.list \
```

Comment:
Have the install instructions changed where a source URL is not needed? But adding the app key is still required? Not sure what's been changed in the parent image hyperledger/sawtooth-shell:nightly .

Reply:
In trying to answer your question, I reverted my change and was surprised the docker-compose ran without the issues, I saw previously. I am not sure why I saw seeing issues previously. I will withdraw my PR.